students/17th/team1/gyuboshim/westagram/user/models.py

```diff
@@ -0,0 +1,11 @@
+from django.db import models
+
+class User(models.Model):
+    phone_number    =   models.CharField(max_length = 15)
+    email_adress    =   models.CharField(max_length = 50)
+    name            =   models.CharField(max_length = 10)
+    nickname        =   models.CharField(max_length = 10)
+    password        =   models.CharField(max_length = 15)
```

Comment:
```suggestion
    phone_number    =   models.CharField(max_length = 15)
    email_adress    =   models.CharField(max_length = 100)
    name            =   models.CharField(max_length = 50)
    nickname        =   models.CharField(max_length = 50)
    password        =   models.CharField(max_length = 300)
```
길이가 너무 적네요. 여유있게 주는게 좋습니다. 현대에 와서는 디스크 가격이 싸기 때문에, 굳이 최소 공간을 적용하는것 보다, 그러한 부분은 로직에서 구별하고, DB자체는 모자라는것보다는 남는게 좋은 쪽으로, 설계해주세요.

Reply:
네 알겠습니다!